clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -49,6 +50,16 @@ const ScanResultsForm = () => {
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const { successAlert } = useAlert();
+
+  useEffect(() => {
+    successAlert(
+      `Your scan was successfully completed, with ${systems.length} new systems detected!`,
+      `Scan Successfully Completed`,
+      { isClosable: true }
+    );
+  }, [successAlert, systems.length]);
```

Comment:
instead of a useEffect, would it work to just call this function after the `upsert` has been successful a few lines down in `confirmRegisterSelectedSystems`? that seems logically more straightforward to me, and might help debug why two successes are showing up

Reply:
This wouldn't show the success alert in the correct spot after the upsert - I would need to move where the alert is being called instead. I'll look into that.